src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -23,6 +23,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private ReadOnly _convKind As ConversionKind
         Private ReadOnly _method As MethodSymbol
 
+        Friend Sub New(kind As ConversionKind)
```

Comment:
Please remove this constructor or add another required parameter for MethodSymbol. We want developers to think what should be passed to the constructor rather than having a convenient way to drop important information.